gslib/tests/test_rm.py

```diff
@@ -745,7 +745,7 @@ def test_rm_multiple_nonexistent_objects(self):
         expected_status=1)
     if self._use_gcloud_storage:
       self.assertIn(
-          'The following URLs matched no objects or files:\n-{}\n-{}'.format(
+          'The following URLs matched no objects or files:\n{}\n-{}'.format(
```

Comment:
This one has another `-`. Is this required?

Reply:
Oops, Missed that! It should be removed.